### PR TITLE
Fix formatting for Munin Plugin Gallery

### PR DESCRIPTION
**Pull request by quotengrote**
Hi,

this PR fixes the formatting for the Munin Plugin Gallery.

cheers
mg

### CODE REVIEW

File: plugins/proxmox/proxmox_vm_count [hunk 1]
```diff
@@ -4,34 +4,35 @@
...
+
+Michael Grote - git.mgrote.net
 
 =head1 LICENSE
 
 GPLv3 or later
 
 SPDX-License-Identifier: GPL-3.0-or-later
 
-=head1 MAGIC MARKERS
```
Comment (sumpfralle) [inline plugins/proxmox/proxmox_vm_count:28]: Currently it is quite well established, that the `MAGIC MARKERS` section (being the least interesting one for human readers) is the *last* one.

I would prefer to keep it like that, if you do not have specific reasons for changing this.

File: plugins/proxmox/proxmox_vm_count [hunk 2]
```diff
@@ -4,34 +4,35 @@
 : << EOF
 =head1 NAME
 
-proxmox_vm_count - Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
+proxmox_vm_count
```
[the remark this answers is not in the document]
Comment (quotengrote) [reply plugins/proxmox/proxmox_vm_count:7]: fixed

File: plugins/proxmox/proxmox_vm_count (hunk 1 above)
Comment (quotengrote) [reply plugins/proxmox/proxmox_vm_count:28]: fixed

**Comment by sumpfralle** [comment]
Thank you for your improvements and sorry for my long delay.

One last thing: you removed the `=cut` marker from these files.
I took a quick look and noticed that `perldoc FILENAME` now produces the documentation *and* all of the following code.
Maybe you were confused by the `EOF` end marker for shell scripts, which are not understood by perldoc?

**Comment by quotengrote** [comment]
So only `cut` is needed instead of `EOF`?

**Comment by quotengrote** [comment]
any news?

**Comment by kenyon** [comment]
@quotengrote did you look at the `perldoc` formatting to be sure it looks good?

**Comment by quotengrote** [comment]
Look good local

**Pull request merged**